benchmark/run.js

```diff
@@ -10,31 +10,56 @@ const cli = CLI(`usage: ./node run.js [options] [--] <category> ...
 
   --filter pattern          string to filter benchmark scripts
   --set    variable=value   set benchmark variable (can be repeated)
+  --format [simple|csv]      optional value that specifies the output format
 `, {
   arrayArgs: ['set']
 });
 const benchmarks = cli.benchmarks();
+const validFormats = ['csv', 'simple'];
 
 if (benchmarks.length === 0) {
-  console.error('no benchmarks found');
-  process.exit(1);
+  console.error('No benchmarks found');
+  // node documentation suggests using this exit method when you need to still
```

Comment:
nit: you are absolutely right. But I don't think it is necessary to say it explicit in a comment.